static/app/utils/replays/frame.tsx

```diff
@@ -25,11 +26,9 @@ export function getColor(frame: ReplayFrame): Color {
       case 'issue':
         return 'red300';
       case 'ui.slowClickDetected':
-        return (frame as SlowClickFrame).data.endReason === 'timeout'
-          ? 'red300'
-          : 'yellow300';
+        return isDeadClick(frame as SlowClickFrame) ? 'red300' : 'yellow300';
```

Comment:
should we add the `isDeadRageClick` check here?

Reply:
nvmnvm they return the same thing

Reply:
`isDeadRageClick` is based on `isDeadClick`, but it's not obvious here. They'll both return the same thing... 
I could write it so it's more obvious.

Reply:
You know what i'll do, there are 4 functions in here with the switch statements repeated over and over, and it's annoying. I'd like to see this file have one switch statement, and return the 4 values from inside each case. So look out for that refactoring sooooon